tests/core/test_core_to_contrib.py

```diff
@@ -84,7 +84,7 @@ def test_is_class_deprecated(self, new_module, old_module):
                     # is used (=3 or =4 if @apply_defaults)
                     assert len(warnings) >= 1
                     assert any(warning.filename == __file__ for warning in warnings)
-                init_mock.assert_called_once_with()
+                init_mock.assert_called_once()
```

Comment:
Not sure if there is a better way to do this while maintain back compatibility.  `DruidCheckOperator` has a default `druid_broker_conn_id` that needs to pass down to `SQLCheckOperator.__init__(conn_id=druid_broker_conn_id, **kwargs)`

Reply:
@xinbinhuang should this help?
```
assert_called_once_with(conn_id=mock.ANY)
```